open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -2525,6 +3183,403 @@ PROCEDURE PRINTPROGRAMINFO
+           ADDNAMELONGNAME=LOAN NAME:LONGNAME
+           ADDNAMEMBRNUMLINK=LOAN NAME:MEMBERNUMLINK
+           ADDNAMEMBRADDRLINK=LOAN NAME:MBRADDRNUMLINK
+           ADDNAMEFIRST=LOAN NAME:FIRST
+           ADDNAMELAST=LOAN NAME:LAST
+           ADDNAMEBIRTHDATE=LOAN NAME:BIRTHDATE
+           ADDNAMESSN=LOAN NAME:SSN
+           ADDNAMESTREET=LOAN NAME:STREET
+           ADDNAMEEXADDR=LOAN NAME:EXTRAADDRESS
+           ADDNAMECITY=LOAN NAME:CITY
+           ADDNAMESTATE=LOAN NAME:STATE
+           ADDNAMEZIPCODE=LOAN NAME:ZIPCODE
+           ADDNAMEHOMEPHONE=LOAN NAME:HOMEPHONE
+           ADDNAMETYPE=LOAN NAME:TYPE
+           ADDNAMELOCATOR=LOAN NAME:LOCATOR
+           ADDNAMESOURCE="LOAN"
+           ADDNAMEADDRESSTYPE=LOAN NAME:ADDRESSTYPE
+           ADDNAMESUBTYPE=LOAN NAME:SUBTYPE
+           ADDNAMENAMEFORMAT=LOAN NAME:NAMEFORMAT
+           ADDNAMESSNTYPE=LOAN NAME:SSNTYPE
+           ADDNAMESSNOVERRIDE=LOAN NAME:SSNOVERRIDE
+           ADDNAMESSNCERTIFICATION=LOAN NAME:SSNCERTIFICATION
+           ADDNAMEFOREIGNTIN=LOAN NAME:FOREIGNTIN
+           ADDNAMEUSPERSONFLAG=LOAN NAME:USPERSONFLAG
+           ADDNAMETITLE=LOAN NAME:TITLE
+           ADDNAMEMIDDLE=LOAN NAME:MIDDLE
+           ADDNAMESUFFIX=LOAN NAME:SUFFIX
+           ADDNAMESEX=LOAN NAME:SEX
+           ADDNAMEPHONETYPE=LOAN NAME:PHONETYPE
+           ADDNAMECARRIERROUTE=LOAN NAME:CARRIERROUTE
+           ADDNAMECOUNTRYCODE=LOAN NAME:COUNTRYCODE
+           ADDNAMECOUNTRY=LOAN NAME:COUNTRY
+           ADDNAMERESTRICT=LOAN NAME:RESTRICT
+           ADDNAMELASTADDRVERIFDATE=LOAN NAME:LASTADDRVERIFDATE
+
+           CALL CHECKFORDUPLICATENAME
+           IF DUPLICATENAMEFOUND=FALSE THEN
+            CALL ADDNAMETOLIST
+
+          END [IF COPYNAMELISTCOUNT<COPYNAMECOUNTMAX]
+
+        END [FOR EACH LOAN NAME]
+      END [IF COPYNAMELISTCOUNT<COPYNAMECOUNTMAX]
+    END [FOR EACH LOAN]
+  END [IF VALUE(SACNAMECOPYLOANFLAG)=TRUE]
+
+ BNODEBUGMSG=FORMAT("End of BUILDCOPYNAMELIST. COPYNAMELISTCOUNT:99",COPYNAMELISTCOUNT)
+ CALL BNODEBUGMSGADD
+
+ FOR TMPLOOP=1 TO COPYNAMELISTCOUNT
+  DO
+   BNODEBUGMSG=FORMAT("99 ",TMPLOOP)+
+               COPYNAMELIST(CNLLOCATOR,TMPLOOP)+"  "+
+               COPYNAMELIST(CNLSHORTNAME,TMPLOOP)+"  Duplicated? "+
+               COPYNAMELIST(CNLDUPLICATED,TMPLOOP)
+   CALL BNODEBUGMSGADD
+  END
+END [PROCEDURE]
+
+PROCEDURE CHECKFORDUPLICATENAME
+[* Check if the name being added to the list is already in the list (by
+** ssn.
+*]
+ DUPLICATENAMEFOUND=FALSE
+
+ BNODEBUGMSG="COPY NAME EVAL "+
+              ADDNAMESOURCE+FORMAT(" NAME LOC #####9",ADDNAMELOCATOR)+
+             "  SSN:"+ADDNAMESSN+
+              FORMAT("  NAME TYPE:9999  ",ADDNAMETYPE)
+
+ IF COPYNAMELISTCOUNT>0 THEN
+  DO
+   FOR TMPLOOP=1 TO COPYNAMELISTCOUNT
+    DO
+     IF ADDNAMESSN=COPYNAMELIST(CNLSSN,TMPLOOP) THEN
+      DO
+       DUPLICATENAMEFOUND=TRUE
+       COPYNAMELIST(CNLDUPLICATED,TMPLOOP)="TRUE"
+      END
+    END
+  END
+
+ BNODEBUGMSG=BNODEBUGMSG+"DUP: "+
```

Comment:
I think you might want a space before the word DUP for the line with BNODEBUGMSG=BNODEBUGMSG+"DUP: "+